### PR TITLE
Updated the JDL page to include new things; plus some order changing

### PR DESCRIPTION
**Pull request by MathieuAA**
Updated the page to include all the latest things about the JDL

### CODE REVIEW

**Comment by MathieuAA** [comment]
The output's not the same between the local version and the one displayed by GitHub. I'll have a PR ready to fix it ASAP. Sorry

File: pages/jdl.md [hunk 1]
```diff
@@ -57,19 +70,33 @@ The entity declaration is done as follows:
...
+entity C {}
+entity D {
+  name String required,
+  address String required maxlength(100),
+  age Integer required min(18)
+}
+```
 
-Here's an example of a field declaration with validations:
+Because the JDL was made to be simple to use and read, f your entity is empty (no field), you can just declare an entity with `entity A` or `entity A {}`.
```
Comment (deepu105) [inline pages/jdl.md]: @MathieuAA does this work in JHUML 1.6.1 ?

File: pages/jdl.md [hunk 2]
```diff
@@ -77,39 +104,59 @@ The relationships declaration is done as follows:
...
+    entity Book
+    entity Author
+    
     relationship OneToMany {
       Author{book} to Book{writer(name)}
     }
 
-### Relationships
-The relationship OneToMany A to B is equivalent to the relationship ManyToOne B to A, you only need make one of them.
+Of course, in real cases, you'd have a lot of relationships and always writing the same three lines could be tedious.
```
Comment (deepu105) [inline pages/jdl.md]: @MathieuAA this as well?

File: pages/jdl.md (hunk 1 above)
Comment (MathieuAA) [inline pages/jdl.md]: no, v1.6.2

File: pages/jdl.md (hunk 2 above)
Comment (MathieuAA) [inline pages/jdl.md]: the same, v1.6.2

**Pull request merged**